tracing/custom-tracing-subscriber/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "custom-tracing-subscriber"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+actix-web = "4.3.1"
+shuttle-actix-web = "0.25.0"
+# disable default features for disabling the Shuttle's default tracing subscriber
```

Comment:
Maybe like this?
```suggestion
# disable default features to disable the Shuttle default tracing subscriber
```

Reply:
6a516838dce5927185d95971109885698e0c1b26